source/mainnet/net/guides/run-local-chain.rst

```diff
@@ -71,7 +71,7 @@ The ``genesis-creator`` tool uses a TOML configuration file format for specifyin
 
 * the initial protocol version
 * cryptographic parameters
-* anonymity revokers
+* anonymity revokers (Identity disclosing authorities)
```

Comment:
Can we replace with `identity disclosing authorities` only?

Reply:
We would need to update the ``genesis-creator`` tool to use a different TOML file configuration. 
```
The ``genesis-creator`` tool uses a TOML configuration file format for specifying parameters from which the genesis data is generated. The TOML file specifies:
...
* anonymity revokers (Identity disclosing authorities)

```